tools/nodejs_api/src_cpp/node_database.h

```diff
@@ -0,0 +1,15 @@
+#include <napi.h>
+#include <iostream>
+#include "main/kuzu.h"
+
+class NodeDatabase : public Napi::ObjectWrap<NodeDatabase> {
+public:
+    static Napi::Object Init(Napi::Env env, Napi::Object exports);
+    NodeDatabase(const Napi::CallbackInfo& info);
+    ~NodeDatabase();
+    friend class NodeConnection;
```

Comment:
why do u need friend class?

Reply:
Because I need to access a kuzu::main::database field in the NodeConnection class (since in the initialization of Connection it takes in the database) and rather than make that field public I think it makes sense to make NodeConnection a friend so that it can access private fields like that.